### PR TITLE
make it possible to deactivate users

### PR DESCRIPTION
**Pull request by tolry**
* deactivated users are removed from
  * votes
  * speakers
* deactivated users remain
  * topic creator
  * comments

Currently there is no admin interface available to activate/deactivate users - but I do not need one right now :smile: 

@smoench @DavidBadura @dragonito any comments, requests, ...?

### CODE REVIEW

**Comment by DavidBadura** [comment]
how can I deactivate an user? can you add a cli command?

**Comment by tolry** [comment]
@DavidBadura good idea, I added two commands for activation and deactivation

**Comment by DavidBadura** [comment]
👍 LGTM

File: src/AppBundle/Command/UserDeactivateCommand.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$username = $input->getArgument('username')) {
```
Comment (smoench) [inline src/AppBundle/Command/UserDeactivateCommand.php]: unused variable `$username`

**Comment by tolry** [comment]
@DavidBadura @dragonito thanks for both your feedback

File: src/AppBundle/Command/UserActivateCommand.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$username = $input->getArgument('username')) {
```
Comment (smoench) [inline src/AppBundle/Command/UserActivateCommand.php]: unused variable `$username`

File: src/AppBundle/Entity/Topic.php [hunk 3]
```diff
@@ -185,7 +185,12 @@ public function getDescription()
      */
     public function getVotes()
     {
-        return $this->votes->toArray();
+        return array_values(array_filter(
```
Comment (smoench) [inline src/AppBundle/Entity/Topic.php]: a `User` type hint should be added and the collection interface could be used:
``` php
return $this->votes
    ->filter(function(User $user) {
        return $user->isActive();
    })
    ->getValues();
```

Comment (tolry) [inline src/AppBundle/Entity/Topic.php]: @smoench right, that would have been more elegant, thx - I will refactor this later

File: src/AppBundle/Command/UserDeactivateCommand.php (hunk 1 above)
Comment (tolry) [inline src/AppBundle/Command/UserDeactivateCommand.php]: yes, thx - this was copy-paste from the user:create command

**Comment by tolry** [comment]
@smoench I made those changes you suggested - thx https://github.com/tolry/talkly/commit/0271c0cd5ca0bf5327ece8510f9e4b94e0884c2a

**Pull request merged**